### PR TITLE
Disable ability to select a date that is in the past

### PR DESCRIPTION
**Pull request by jbforth4**
Added js function to only allow the current date and future dates to be selected in the pickupEditCreate folder.  Also updated the html file to use the js function.

### CODE REVIEW

File: client/app/components/group/_pickupEditCreate/pickupEditCreate.controller.js [hunk 1]
```diff
@@ -137,4 +137,14 @@ class pickupEditCreateController {
   }
 }
 
+  noPastDateCalender() {
+  this.myDate = new Date();
```
Comment (tiltec) [inline client/app/components/group/_pickupEditCreate/pickupEditCreate.controller.js]: better call this `now`?
Also, you just use this in local scope, so you don't need to define it on `this`.

File: client/app/components/group/_pickupEditCreate/pickupEditCreate.controller.js [hunk 2]
```diff
@@ -137,4 +137,14 @@ class pickupEditCreateController {
   }
 }
 
+  noPastDateCalender() {
```
Comment (tiltec) [inline client/app/components/group/_pickupEditCreate/pickupEditCreate.controller.js]: maybe rather 'setMinDate'? It describes more accurately what the code is doing.

Also, you need to invoke this method somewhere. I'd recommend the $onInit method (just append it)

**Comment by tiltec** [comment]
Does it make sense to you? It doesn't work, can you guess why?

**Comment by jbforth4** [comment]
After the changes you made I was able to compile but the test environment
still won't launch. I was up messing around with it last night so without
being able to test and see the errors it it's hard to guess

**Comment by tiltec** [comment]
Solved via #517

**Pull request closed without merging**